hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -111,10 +111,14 @@
    */
   public ContainerBalancerConfiguration(
       OzoneConfiguration ozoneConfiguration) {
-    this.ozoneConfiguration = ozoneConfiguration;
+    if (ozoneConfiguration == null) {
```

Comment:
actually, ozoneConfiguration should not be null. we'd better to  use `Preconditions.checkNotNull` instead here.